src/libs/gif/lv_gif.c

```diff
@@ -137,6 +137,36 @@ void lv_gif_resume(lv_obj_t * obj)
     lv_timer_resume(gifobj->timer);
 }
 
+bool lv_gif_loaded(lv_obj_t * obj)
+{
+    lv_gif_t * gifobj = (lv_gif_t *) obj;
+
+    return (gifobj->gif != NULL);
+}
+
+int lv_gif_get_loop_count(lv_obj_t * obj)
```

Comment:
Use `int32_t` type instead of int, consistent with loop_count data type.

Reply:
Solved with a fixup, thanks